ui/app/AppLayouts/Chat/CommunityComponents/CreateCommunityPopup.qml

```diff
@@ -23,334 +28,337 @@ ModalPopup {
 
-                StatusIconButton {
-                    icon.name: "caret"
-                    iconRotation: -90
-                    iconColor: Style.current.textColor
-                    icon.width: 13
-                    icon.height: 7
-                    anchors.right: parent.right
-                    anchors.rightMargin: Style.current.smallPadding
-                    anchors.top: parent.top
-                    anchors.topMargin: colorPicker.textField.height / 2 - height / 2 + Style.current.bigPadding
-                    onClicked: colorDialog.open()
-                }
+                    bgColor: colorDialog.colorSelected ? 
+                        colorDialog.color : Theme.palette.baseColor2
+                    contentColor: colorDialog.colorSelected ? Theme.palette.indirectColor1 : Theme.palette.baseColor1
+                    text: colorDialog.colorSelected ?
+                        colorDialog.color.toString().toUpperCase() : 
+                        qsTr("Pick a color")
 
-                ColorDialog {
-                    id: colorDialog
-                    title: qsTr("Please choose a color")
-                    color: colorPicker.defaultColor
-                    onAccepted: {
-                        colorPicker.text = colorDialog.color
+                    onClicked: colorDialog.open();
+                    onTextChanged: {
+                        if (colorDialog.colorSelected) {
+                            validationError = Utils.validateAndReturnError(text, communityColorValidator)
+                        }
                     }
+
+                    ColorDialog {
+                        id: colorDialog
+                        property bool colorSelected: false
+                        color: Theme.palette.primaryColor1
+                        onAccepted: colorSelected = true
+                    }
+                }
+
+                StatusBaseText {
+                    text: colorSelectorButton.validationError
+                    visible: !!text
+                    color: Theme.palette.dangerColor1
+                    anchors.top: colorSelectorButton.bottom
+                    anchors.topMargin: 4
+                    anchors.right: colorSelectorButton.right
                 }
             }
 
-            Separator {
-                id: separator1
-                anchors.top: colorPicker.bottom
-                anchors.topMargin: isEdit ? 0 : Style.current.bigPadding
-                anchors.left: parent.left
-                anchors.leftMargin: -Style.current.padding
-                anchors.right: parent.right
-                anchors.rightMargin: -Style.current.padding
+            StatusModalDivider {
+                topPadding: 8
+                bottomPadding: 8
                 visible: !isEdit
             }
 
-            StatusSettingsLineButton {
-                id: membershipRequirementSetting
-                // TODO: remove 'isEnabled: false' when we no longer need to force
```

Comment:
I don't know what this comment is about. Especially the 'force' part.

So this removal most likely needs review. The equivalent on the right hand-side is only set to invisible when editing a community. There's no such thing as being enabled/disabled.

Can someone give me some insights on what this is about here?